features/step_definitions/administration.rb

```diff
@@ -1,99 +1,110 @@
 Given(/^I have an admin account$/) do
-  pending # express the regexp above with the code you wish you had
+ @admin = AdminUser.create(email: "admin@example.com", password: "password", 
+    password_confirmation: "password")
 end
 
 When(/^I visit the admin panel url$/) do
-  pending # express the regexp above with the code you wish you had
+  visit("/admin") 
 end
 
 When(/^I enter my admin email$/) do
-  pending # express the regexp above with the code you wish you had
+  fill_in :admin_user_email, with: "admin@example.com" 
```

Comment:
It doesn't really matter, but I would stick with using parenthesis for the method calls here, for consistency.